src/screencast/view.ts

```diff
@@ -39,8 +40,8 @@ export class ScreencastView {
                   <i class="codicon codicon-refresh"></i>
               </button>
               <input id="url" />
-              <button id="inspect" title="Open DevTools">
-                  <i class="codicon codicon-inspect"></i>
+              <button id="inspect" title="Toggle DevTools">
```

Comment:
Nit: Should we have this switch between "Open DevTools" and "Close DevTools" based on state (since we have it right here)?